torch/_dynamo/optimizations/torchxla_integration.py

```diff
@@ -97,9 +204,11 @@ def extract_compiled_graph(xla_model: torch.fx.GraphModule, xla_args):
     ]
 
     if debug:
+        print(f"Graph module:\n{xla_model.code}")
         print(f"args_tensor_ids {args_tensor_ids}")
 
     tensor_id_to_arg_idx = {tensor_id: i for i, tensor_id in enumerate(args_tensor_ids)}
+    metrics.clear_counters()
```

Comment:
why do we need to clear counters here?

Reply:
`get_fallback_ops` below use counter to detect torch_xla fallbacks.
Clear the counters here so we only detect fallbacks happening when running
the `xla_model` below.

I'll add the above to the comment